mocks/auth.js

```diff
@@ -35,6 +38,15 @@ class FakeAuth {
     return Promise.resolve(this.currentUserRecord);
   }
 
+  getUser() {
+    return Promise.resolve(mockGetUser(...arguments));
```

Comment:
I like this pattern, which [allows you to do `mockGetUser.mockReturnValue`](https://github.com/Upstatement/firestore-jest-mock/pull/36/files#diff-445ddbc16541e58a306f94edf4461b34R122)

However, if a user doesn't do that, but has an assertion that relies on it being an object, would a test pass?  Would it work if we did:
`return Promise.resolve(mockGetUser(...arguments) || {})`?

Reply:
It should. I think I do that in other places but neglected that here.